src/main/resources/tags/links/uuid.tag

```diff
@@ -2,5 +2,16 @@ type: text
 aliases: xuid, floodgateuuid, bedrockuuid
 
 ---
+Automatic method:
+Use https://uuid.kejona.dev/ to convert any Bedrock Player username into a Floodgate UUID.
+Alternatively, run the "/UUID" command in #botspam.
+Manual method:
+You can manually find out a UUID using this method:
+Step 1: Visit https://www.cxkes.me/xbox/xuid and enter the bedrock username (without the period that is added by Geyser)
+Step 2: Search for the hexidecimal result and copy it down
+Step 3: You can try just that number instead of the username, but mine was formatted like this
```

Comment:
This wording, bluntly, sounds like a blog when we know what will happen. Every output of that site should need the conversion from below.

Reply:
fixed in the latest commit